3_summarize.yml

```diff
@@ -63,6 +63,7 @@ targets:
       final_target = target_name,
       site_files = pgdl_site_files,
       ice_files = ice_files,
+      n_cores = 1,
```

Comment:
That is the PGDL one and I haven't tested those yet. I focused on PB0 since Hayley was finishing up PGDL runs. I wanted to do the same process where I tried just 4 lakes first before kicking off the full build.

Reply:
Gotcha. :+1: